robottelo/ui/base.py

```diff
@@ -539,11 +540,14 @@ def click(self, locator, wait_for_ajax=True,
         :raise: UINoSuchElementError if the element could not be found.
 
         """
-        element = self.wait_until_element(locator, timeout=waiter_timeout)
+        if type(locator).__name__ == 'tuple':
+            element = self.wait_until_element(locator, timeout=waiter_timeout)
+        else:
+            element = locator
         if element is None:
             raise UINoSuchElementError(
-                '{0}: element with locator {1} not found while trying to click'
-                .format(type(self).__name__, locator)
+                '{0}: element was not found while trying to click'
```

Comment:
We can try to get more information from the element, like class and id attributes and the tag name this can at give more information.

Also what about adding a flag and if it is a locator print the locator as before? Here can also be used `element == locator` this is gonna be true when a WebElement is passed.